### ISSUE TITLE
 Julia v1.1.0 - Installation Error

### ISSUE THREAD

**Issue by felipemarkson** [dss-extensions/OpenDSSDirect.jl#45]
Only the development version can be installed.

When I try to install the stable version:
```
julia> VERSION
v"1.1.0"
julia> Pkg.add("OpenDSSDirect")
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package OpenDSSDirect [a8b11937]:
 OpenDSSDirect [a8b11937] log:
 ├─possible versions are: [0.1.0-0.1.1, 0.2.0, 0.3.1-0.3.2, 0.4.0] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0-0.1.1, 0.2.0, 0.3.1-0.3.2, 0.4.0]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions
 ```

**Comment by kdheepak** [comment]
It hasn't been merged into METADATA.jl yet. See https://github.com/JuliaLang/METADATA.jl/pull/21550. Once that is merged you'll be able to install the stable.

You can do

```
pkg> add https://github.com/dss-extensions/OpenDSSDirect.jl#v0.5.1
```

to install the stable if you like.

**Comment by felipemarkson** [comment]
Thank you
